docs/tutorials/custom_contracts/fabric.md

```diff
@@ -0,0 +1,731 @@
+            "type": "string"
+          }
+        },
+        {
+          "name": "value",
+          "schema": {
+            "type": "string"
+          }
+        }
+      ],
+      "returns": []
+    }
+  ],
+  "events": [
+    {
+      "id": "27564533-30bd-4536-884e-02e5d79ec238",
+      "interface": "f1e5522c-59a5-4787-bbfd-89975e5b0954",
+      "namespace": "default",
+      "pathname": "AssetCreated",
+      "signature": "",
+      "name": "AssetCreated",
+      "description": "",
+      "params": null
+    }
+  ]
+}
+```
+
+## Create an HTTP API for the contract
+
+Now comes the fun part where we see some of the powerful, developer-friendly features of FireFly. The next thing we're going to do is tell FireFly to build an HTTP API for this chaincode, complete with an OpenAPI Specification and Swagger UI. As part of this, we'll also tell FireFly where the chaincode is on the blockchain.
+
+Like the interface broadcast above, this will also generate a broadcast which will be pinned to the blockchain so all the members of the network will be aware of and able to interact with this API.
+
+We need to copy the `id` field we got in the response from the previous step to the `interface.id` field in the request body below. We will also pick a name that will be part of the URL for our HTTP API, so be sure to pick a name that is URL friendly. In this case we'll call it `asset_transfer`. Lastly, in the `location` field, we're telling FireFly where an instance of the chaincode is deployed on-chain, which is a chaincode named `asset_transfer` in the channel `firefly`.
+
+> **NOTE**: The `location` field is optional here, but if it is omitted, it will be required in every request to invoke or query the chaincode. This can be useful if you have multiple instances of the same chaincode deployed to different channels.
+
+### Request
+
+`POST` `http://localhost:5000/api/v1/namespaces/default/apis`
+
+```json
+{
+  "name": "asset_transfer",
+  "interface": {
+    "id": "f1e5522c-59a5-4787-bbfd-89975e5b0954"
+  },
+  "location": {
+    "channel": "firefly",
+    "chaincode": "asset_transfer"
+  }
+}
+```
+
+### Response
+
+```json
+{
+  "id": "a9a9ab4e-2544-45d5-8824-3c05074fbf75",
+  "namespace": "default",
+  "interface": {
+    "id": "f1e5522c-59a5-4787-bbfd-89975e5b0954"
+  },
+  "location": {
+    "channel": "firefly",
+    "chaincode": "asset_transfer"
+  },
+  "name": "asset_transfer",
+  "message": "5f1556a1-5cb1-4bc6-8611-d8f88ccf9c30",
+  "urls": {
+    "openapi": "http://127.0.0.1:5000/api/v1/namespaces/default/apis/asset_transfer/api/swagger.json",
+    "ui": "http://127.0.0.1:5000/api/v1/namespaces/default/apis/asset_transfer/api"
+  }
+}
+```
+
+## View OpenAPI spec for the contract
+
+You'll notice in the response body that there are a couple of URLs near the bottom. If you navigate to the one labeled `ui` in your browser, you should see the Swagger UI for your chaincode.
+
+![Swagger UI](../images/simple_storage_swagger.png "Swagger UI")
```

Comment:
The path here needs to have another `../` in it. Also, this currently points to the swagger UI for Simple Storage which is not the contract that we're working with in this guide